schema.d.ts

```diff
@@ -372,6 +374,29 @@ export interface CssPart {
   deprecated?: boolean | string;
 }
 
+/**
+ * The description of a CSS Custom State
+ */
+export interface CssCustomState {
+  name: string;
```

Comment:
Note unlike custom properties and unlike the WICG proposal the standardised custom state won't require a dashed ident name. The standard uses `:state(<ident>)` syntax rather than `:<dashed-ident>`

Reply:
We should probably describe that in the jsdoc for the name field then.